opium/src/body.ml

```diff
@@ -1 +1,51 @@
 include Rock.Body
+
+let log_src = Logs.Src.create "opium.body.of_file"
+
+module Log = (val Logs.src_log log_src : Logs.LOG)
+
+exception Isnt_a_file
+
+let of_file ?(local_path = "") fname =
```

Comment:
I would remove `local_path` entirely, I don't think it makes sense to keep it if the function is exposed in the API

Reply:
Fixed